charts/consul/templates/connect-inject-deployment.yaml

```diff
@@ -193,6 +193,26 @@ spec:
                 -consul-sidecar-cpu-request={{ $consulSidecarResources.requests.cpu }} \
                 {{- end }}
                 {{- end }}
+          livenessProbe:
+            httpGet:
+              path: /readyz/ready
+              port: 9445
+              scheme: HTTP
+            failureThreshold: 2
+            initialDelaySeconds: 1
+            periodSeconds: 2
```

Comment:
I'm curious if it needs to be checked every 2 seconds? How likely is it for the certs to be removed from disk?

Reply:
Yep, we'll take another look. We've added the same configuration as before the refactor, but we will re-think whether those still make sense.

Reply:
We kept periodSeconds=2, as it was the value we had before the t-proxy refactor; we looked more into Kubernetes documentation, and the default value for periodSeconds is 10. Do you have any guidance as to whether we should increase the value, and by how much? And to your point once there the certificates are available, they are unlikely to be removed.